.travis.yml

```diff
@@ -1,5 +1,20 @@
+sudo: required
+dist: trusty
+
 language: ruby
 rvm:
   - 2.0
+
+env:
+  - TRAVIS_NODE_VERSION="6"
+
+install:
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
+  - npm install -g writewell
+
 script:
   - script/cibuild
+  - cd /_site
+  - find . -type f -name "*.html" >> ./script/list-of-files.txt
+  - cd ..
+  - npm test
```

Comment:
@stoe I think one of us read the other's mind, I just committed that in the most recent commit! 👍 😄